### PR TITLE
Fix/class name

### PR DESCRIPTION
**Pull request by Crash--**
class is not a valid react attribute. className is

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -10409,6 +10409,7 @@ minilog@3.1.0:
 "minilog@https://github.com/cozy/minilog.git#master":
   version "3.1.0"
   resolved "https://github.com/cozy/minilog.git#f01f7d9dfe20981177dd34b9662c2f077d818f82"
+  integrity sha1-0tDxiHyjY9Gs8OqG1cTfKTs/tnU=
```
Comment (y-lohse) [inline yarn.lock]: Why is this change here?

Comment (Crash--) [reply yarn.lock]: Because we merged a PR without editing this file in fact. This is needed ;) 

I closed the PR fixing this issue this morning this I've also the commit in this branch.
